jarviscli/plugins/file_manager.py

```diff
@@ -0,0 +1,104 @@
+from plugin import plugin
+import os
+import shutil
+
+@plugin("file manage")
+class file_manage:
+    def __call__(self,jarvis,s):
+        self.get_file_directory(jarvis)
+        self.get_cmd(jarvis)
+
+        if self.cmd == "delete":
+            self.delete(jarvis,self.file)
+        elif self.cmd == "move":
+            self.move(jarvis,self.file)
+        elif self.cmd == "rename":
+            self.rename(jarvis,self.file)
+
+    def get_file_directory(self,jarvis):
+        self.file = jarvis.input("Enter the directory of the file you would like to edit: ")
+
+    def get_cmd(self,jarvis):
+        # function to find command to be performed to file
+
+        self.possibleCmds = ["delete","move","rename"]
+
+        cmdValid = False
+        while not cmdValid:
+            # iterate through possible commands and say each
+            jarvis.say("Commands Avaliable")
+
+            i = 1
+            for cmd in self.possibleCmds:
+                jarvis.say(str(i) + ". " + cmd)
+                i = i + 1
+
+            self.cmd = jarvis.input("Enter command to be performed: ")
```

Comment:
Minor hint: You don't need to store ``cmd`` in ``self`` - you could also return ``cmd`` at end of this method. Usually if there is no benefit from string stuff as class-variable, you should prefer function return.

Imagine for example hypothetical Jarvis gets a multi-user interface and two users use your file manager same time. Then this approach could fail. But really nothing wrong with your code, just a general hint.